src/ws/ws_manager.rs

```diff
@@ -171,10 +170,14 @@ impl WsManager {
                 .map_err(|e| Error::JsonParse(e.to_string()))?
         {
             "userEvents".to_string()
+        } else if let Subscription::OrderUpdates { user: _ } =
```

Comment:
This is duplicative with :227, but I didn't want to go into a whole refactoring of the file. I think there is a better way to manage the user based subscriptions than the magic string.

Reply:
I think that's fine, and agreed, the identifier should be an enum instead